src/condor_daemon_core.V6/condor_timer_manager.h

```diff
@@ -68,17 +66,16 @@ typedef void    (Service::*Releasecpp)(void*);
 #define TimerReleasecpp Releasecpp
 
 // This value, passed for "when", will cause the timer to never expire
-const unsigned	TIMER_NEVER = 0xffffffff;
-
-const time_t TIME_T_NEVER	= 0x7fffffff;
+const time_t TIMER_NEVER	 = -1;
+const time_t TIME_T_NEVER	= -1;
```

Comment:
```suggestion
const time_t TIMER_NEVER	 = std::numeric_limits<time_t>::max();
const time_t TIME_T_NEVER	= std::numeric_limits<time_t>::max();
```
These need to be positive values.